src/drawing.c

```diff
@@ -153,23 +185,33 @@ void coord_list_free (GromitData *data,
   devdata->coordlist = NULL;
 }
 
+void cleanup_context(GromitPaintContext *context)
```

Comment:
That sounds like it's free()-ing the whole context ;-) Maybe clear_context and some comment?

Reply:
Actually, just inlining this would be most clear, I think.